config/clients/java/template/Java/StringUtil.mustache

```diff
@@ -0,0 +1,72 @@
+{{>licenseInfo}}
+
+package {{invokerPackage}};
+
+import java.util.Collection;
+import java.util.Iterator;
+
+{{>generatedAnnotation}}
+public class StringUtil {
```

Comment:
Note to self: Can this be lowered visibility? Package private or protected maybe?

Slightly different `StringUtil` and `StringUtils` classes with similar methods from various projects (Apache Commons, Spring, AWS SDK, ATG, etc) are a common cause of weird bugs